.github/workflows/trivy-scan.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - name: Set up Go 1.x
         uses: actions/setup-go@v1
         with:
-          go-version: "1.17"
+          go-version: "1.18"
```

Comment:
If we are using a specific version of go in kind-action we should use it here as well. If not we might be running scans on different versions of the stdlib